react-hs-examples/src/TodoComponents.hs

```diff
@@ -56,7 +56,7 @@ todoTextInput = mkStatefulView "todo text input" "" $ \curText args ->
              if keyCode evt == 13 && not (T.null curState) -- 13 is enter
                  then (tiaOnSave args curState, Just "")
                  else ([], Nothing)
-        , onFocus $ \_ _ -> simpleHandler $ \curState ->
```

Comment:
you could also keep the more informative `curState`, but prefix with a `_`, i.e. make it `_curState`.  that will keep explaining things to the reader while silencing the warning.